project/Http4sPlugin.scala

```diff
@@ -300,7 +301,7 @@ object Http4sPlugin extends AutoPlugin {
   lazy val gatlingHighCharts                = "io.gatling.highcharts"  %  "gatling-charts-highcharts" % gatlingTest.revision
   lazy val http4sWebsocket                  = "org.http4s"             %% "http4s-websocket"          % "0.2.1"
   lazy val javaxServletApi                  = "javax.servlet"          %  "javax.servlet-api"         % "3.1.0"
-  lazy val jawnJson4s                       = "org.spire-math"         %% "jawn-json4s"               % "0.12.1"
+  lazy val jawnJson4s                       = "org.spire-math"         %% "jawn-json4s"               % "0.12.2"
```

Comment:
There's 0.13.0 now, which looks the same as 0.12.2, but would require a jawn-fs2 bump to avoid warnings.

This compiled agianst json4s-3.5, and we'll need to bump that to get to 3.6. :weary: